### ISSUE TITLE
iPad support

### ISSUE THREAD

**Issue by xmollv** [xmollv/Llitgi#23]
It would be pretty cool to have support for the iPad. The idea that I have for decent iPad support would be something like this:

- [ ] Add a `UISplitViewController` to have the master/detail behaviour. On the detail, we would have the `SFSafariViewController`. This would also require to add an empty state for the detail.
- [ ] Make sure that all the text displayed on the app follows the `readable guide`.
- [ ] The Settings screen should be presented as a `.formSheet` or as a `.pageSheet`.
- [ ] Add the app icons for the iPad.

I ran a quick test to see if it's possible to achieve my idea and it seems so. It's completely broken because the `tabBar` is not showing the icons nor the titles of the `UIViewControllers` and the detail isn't working, but it helps to illustrate my idea.

![Demo](https://user-images.githubusercontent.com/8229382/43035026-170c2e9a-8ce8-11e8-870d-2309548d5871.gif)

**Comment by igorcferreira** [comment]
I believe that the UITabBar should not be present over the split view controller. Instead, it should be restrained to the master view controller. This way, there is no double tab when presenting the safari view controller.

Ref.: https://stackoverflow.com/questions/33668952/uitabbarcontroller-inside-master-of-uisplitviewcontroller

**Comment by FWDavide** [comment]
First we need dark-mode #22 .. otherwise it will be unreadable on iPad

**Comment by xmollv** [comment]
@igorcferreira I completely agree on it, I didn't think about the double bar on the detail!

**Comment by atineoSE** [comment]
For what is worth, I find the white colour scheme great and perfectly readable on iPad. Having dark mode doesn't hurt, of course, but I don't think it should hold back the release of iPad support.

**Comment by xmollv** [comment]
@atineoSE This is the crash mentioned:

<img width="1440" alt="screen shot 2018-08-07 at 22 34 42" src="https://user-images.githubusercontent.com/8229382/43801245-cdaa8e1e-9a92-11e8-99af-505585161d13.png">

That's from this current implementation (just pulled your changes). The same happens to me in another branch that I'm working on. Forget about it, I might have a new PR with your changes (that I've been picking) + coordinator that might render this one obsolete. Don't feel forced to spend any more time with it unless you're doing it for fun, I'll push my branch ASAP. (I had to repost it because I used my company account, woops 😅)
